designer/server/src/test/scala/pl/touk/nussknacker/ui/api/ExpressionSuggesterSpec.scala

```diff
@@ -180,14 +189,15 @@ class ExpressionSuggesterSpec extends AnyFunSuite with Matchers with PatientScal
       "#input",
       "#listOfUnions",
       "#listVar",
+      s"#${VariableConstants.MetaVariableName}",
```

Comment:
This is just a suggestion, but I prefer when all the data in test assertions are literal.

Reply:
But know someone who reads it can figure out why this list contains also variable that is not in included in a set of input (given) variables